### PR TITLE
Implemented: 'Learn More' functionality that generates job info from GitBook Lens APIs (#726)

### PR DESCRIPTION
**Pull request by R-Sourabh**
### Related Issues

#726 

### Short Description and Why It's Useful

- Added an `ion-item` with `jobEnumId` and an icon to the job card.
- Added the text "Learn more" next to the job description in the job configuration section.
- A modal now opens when clicking either the icon or the "Learn more" text.
- The "Learn more" modal displays job information generated from the GitBook Lens API and includes a link to the GitBook article source.
### Screenshots of Visual Changes before/after (If There Are Any)

Screenshot from 2024-10-08 16-11-22
Screenshot from 2024-10-08 16-11-14

### Contribution and Currently Important Rules Acceptance

- [x] I read and followed [contribution rules](https://github.com/hotwax/job-manager#contribution-guideline)

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -18,3 +18,7 @@ VUE_APP_ALIAS=
 VUE_APP_DEFAULT_LOG_LEVEL="error"
 VUE_APP_LOGIN_URL="http://launchpad.hotwax.io/login"
 VUE_APP_BROKER_JOB_ENUMS = {"REJ_ORDR":"JOB_BKR_REJ_ORD"}
+VUE_APP_GITBOOK_API_KEY="gb_api_hL97xDJyiI6FBhnWKdm8CP6N6Wjl5nZSjhK3Nyd7"
```
Comment (dt2patel) [inline .env.example:21]: should we be committing this to our repo?

Comment (R-Sourabh) [reply .env.example:21]: No, sir, we shouldn't. I have reverted the changes.

File: src/components/LearnMoreModal.vue [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+        } else {
+          throw resp.data;
+        }
+      } catch(error: any) {
+        logger.error(error);
+        this.isGeneratingAnswer = false;
+      }
+    },
+    async redirectToDoc(section: any) {
+      window.open(`https://docs.hotwax.co/documents/retail-operations/${section.path}`, "_blank")
```
Comment (ymaheshwari1) [inline src/components/LearnMoreModal.vue:153]: Pass noopener, noreferrer when opening link

File: src/components/LearnMoreModal.vue [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+    },
+    async redirectToJobsDoc() {
```
Comment (ymaheshwari1) [inline src/components/LearnMoreModal.vue:155]: Same as above

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

**Pull request merged**